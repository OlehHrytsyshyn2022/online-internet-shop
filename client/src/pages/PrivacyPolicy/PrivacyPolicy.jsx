import React from "react";
import { Typography, Box } from "@mui/material";
import { useMediaQuery } from "@mui/material";

const PrivacyPolicy = () => {
  const isMobile = useMediaQuery("(max-width: 670px)");

  const fontSizeTitle = isMobile ? "20px" : "33px";
  const fontWeightTitle = isMobile ? 400 : 500;
  const textAlignTitle = isMobile ? "start" : "center";

  const fontSize = isMobile ? "16px" : "26px";

  const fontSizeBody = isMobile ? "12px" : "16px";

  return (
    <div
      style={{
        margin: "0 auto",
        padding: "40px 16px 0",
        fontWeight: 400,
        maxWidth: "670px",
      }}
    >
      <Box sx={{ textAlign: textAlignTitle }}>
        <Typography
          variant="h1"
          sx={{
            fontSize: fontSizeTitle,
            fontWeight: fontWeightTitle,
          }}
        >
          Privacy Policy
        </Typography>
      </Box>
      <Typography
        variant="body1"
        sx={{ fontSize: fontSizeBody, paddingTop: "40px" }}
      >
        We understand the importance of your privacy and are committed to
        protecting your personal data. This document describes how we collect,
        use, process, and safeguard your information.
      </Typography>
      <Typography
        variant="h2"
        sx={{
          fontSize: fontSize,
          fontWeight: 400,
          paddingTop: "22px",
        }}
      >
        Security
      </Typography>
      <Typography
        variant="body1"
        sx={{ fontSize: fontSizeBody, paddingTop: "14px" }}
      >
        We make every effort to ensure the security of your information. We
        utilize advanced technologies and security measures to protect your data
        from unauthorized access and use.
      </Typography>
      <Typography
        variant="h2"
        sx={{
          fontSize: fontSize,
          fontWeight: 400,
          paddingTop: "22px",
        }}
      >
        Cookies
      </Typography>

      <Typography sx={{ fontSize: fontSizeBody, padding: "10px" }}>
        ● Duis rutrum dictum libero quis rutrum. Etiam sed neque aliquam,
        sollicitudin ante a, gravida arcume
      </Typography>
      <Typography sx={{ fontSize: fontSizeBody, padding: "10px" }}>
        ● We use cookies to enhance the functionality of our website. They help
        us provide a convenient online experience for you.
      </Typography>
      <Typography sx={{ fontSize: fontSizeBody, padding: "10px" }}>
        ● Nam fringilla molestie velit, eget pellentesque risus scelerisque
      </Typography>
      <Typography sx={{ fontSize: fontSizeBody, padding: "10px" }}>
        ● We also use cookies to analyze website usage data and improve its
        functionality.
      </Typography>
    </div>
  );
};

export default PrivacyPolicy;
