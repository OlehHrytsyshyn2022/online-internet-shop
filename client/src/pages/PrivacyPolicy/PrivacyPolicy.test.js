import React from "react";
import { render } from "@testing-library/react";
import PrivacyPolicy from "./PrivacyPolicy";

describe("PrivacyPolicy", () => {
  test("renders PrivacyPolicy page", () => {
    const { getByText } = render(<PrivacyPolicy />);
    const AboutText = getByText("Privacy Policy");
    expect(AboutText).toBeInTheDocument();
  });

  test("Snapshot Test PrivacyPolicy", () => {
    const { container } = render(<PrivacyPolicy />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
