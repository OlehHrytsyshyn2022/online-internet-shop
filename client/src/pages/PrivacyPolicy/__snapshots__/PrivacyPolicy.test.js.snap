// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivacyPolicy Snapshot Test PrivacyPolicy 1`] = `
<div
  style="margin: 0px auto; padding: 40px 16px 0px; font-weight: 400; max-width: 670px;"
>
  <div
    class="MuiBox-root css-xi606m"
  >
    <h1
      class="MuiTypography-root MuiTypography-h1 css-1k1uezg-MuiTypography-root"
    >
      Privacy Policy
    </h1>
  </div>
  <p
    class="MuiTypography-root MuiTypography-body1 css-1r2tuf8-MuiTypography-root"
  >
    We understand the importance of your privacy and are committed to protecting your personal data. This document describes how we collect, use, process, and safeguard your information.
  </p>
  <h2
    class="MuiTypography-root MuiTypography-h2 css-s294ip-MuiTypography-root"
  >
    Security
  </h2>
  <p
    class="MuiTypography-root MuiTypography-body1 css-48hir6-MuiTypography-root"
  >
    We make every effort to ensure the security of your information. We utilize advanced technologies and security measures to protect your data from unauthorized access and use.
  </p>
  <h2
    class="MuiTypography-root MuiTypography-h2 css-s294ip-MuiTypography-root"
  >
    Cookies
  </h2>
  <p
    class="MuiTypography-root MuiTypography-body1 css-1uiqyha-MuiTypography-root"
  >
    ● Duis rutrum dictum libero quis rutrum. Etiam sed neque aliquam, sollicitudin ante a, gravida arcume
  </p>
  <p
    class="MuiTypography-root MuiTypography-body1 css-1uiqyha-MuiTypography-root"
  >
    ● We use cookies to enhance the functionality of our website. They help us provide a convenient online experience for you.
  </p>
  <p
    class="MuiTypography-root MuiTypography-body1 css-1uiqyha-MuiTypography-root"
  >
    ● Nam fringilla molestie velit, eget pellentesque risus scelerisque
  </p>
  <p
    class="MuiTypography-root MuiTypography-body1 css-1uiqyha-MuiTypography-root"
  >
    ● We also use cookies to analyze website usage data and improve its functionality.
  </p>
</div>
`;
